vendors/infineon/boards/xmc4800_iotkit/ports/wifi/iot_wifi.c

```diff
@@ -432,6 +433,8 @@ WIFIReturnCode_t WIFI_Ping( uint8_t * pucIPAddr,
     status = eWiFiTimeout;
   }
 
+  ESP_CFG_DBG_OUT( "WIFI_Ping returning %d \r\n", ret );
```

Comment:
Nit: Should this be in the failure path? Will this clutter the logs for the happy path of this API?